tests/basic.lua

```diff
@@ -172,6 +172,48 @@ for _it, a in iter(1, 2, 3, 4, 5, 6, 7) do print(a) end
 error: object 1 of type "number" is not iterable
 --test]]
 
+--------------------------------------------------------------------------------
+-- pairs
+--------------------------------------------------------------------------------
+
+for _it, a in fun.pairs({4, 5, 6}) do print(a) end
```

Comment:
Not sure if I really get why somebody would need this fun.pairs().

Reply:
It is rather looks like `fun.values`, since it ignores indexes and keys.

Reply:
It wraps lua's `pairs` function so it's named the same way. When one read's `fun.pairs` he can easily decide that it works the same way as vanilla `pairs` and has same corner cases. With `values`  he need to read docs to be sure.

It was made similar to all other  fun methods and make this method safe replacement to `fun.iter`, so it provides single argument in chained functions. Maybe it's better to remove current implementation and rename this method to `values` and `map_pairs` to  `pairs`. I like both approaches, we can ask somebody else for advice/suggestion.

Reply:
But it does not work the same as the original pairs. It returns only values. The original pairs returns keys too. `fun.map_pairs` works like the original pairs. `fun.pairs` works like I don't know what. So I still don't understand, why `fun.pairs` is named `pairs`.

Talking of `fun.iter` removal - I don't think we can do that. We will break existing users. If it can't be fixed, then let it be as is.

Reply:
I'm not saying about `iter` removal. It can be neither fixed nor removed in a backward-compatible way.

Reply:
Oh, I thought that `Maybe it's better to remove current implementation` was about `fun.iters`. Ok then.